packages/web/src/pages/audio-rewards-page/components/SendInputConfirmation.tsx

```diff
@@ -40,6 +49,20 @@ const SendInputConfirmation = ({
   recipientAddress,
   onSend
 }: SendInputConfirmationProps) => {
+  const [hasOneSecondElapsed, setHasOneSecondElapsed] = useState(false)
```

Comment:
nit: best if we don't name the variable after the duration itself, so we give ourselves some flexibility to change the duration as config w/o changing the code. Something like `hasLoadingDurationElapsed`, idk